va/dns_account_test.go

```diff
@@ -89,16 +88,17 @@ func TestDNSAccount01Validation(t *testing.T) {
 			va, _ := setup(nil, "", nil, nil)
 			_, err := va.validateDNSAccount01(ctx, tc.ident, expectedKeyAuthorization, testAccountURI)
 
-			if tc.wantErr {
+			if tc.wantErrMsg != "" {
 				if err == nil {
-					t.Errorf("validateDNSAccount01(%q) = success, but want error", tc.ident.Value)
+					t.Errorf("validateDNSAccount01(%q) = success, but want error %q", tc.ident.Value, tc.wantErrMsg)
 					return
```

Comment:
Sorry I didn't notice this last time! You can use `t.Fatalf` instead of `t.Errorf; return`. This makes it harder to accidentally break the test by removing a bare return in a later refactoring.
```suggestion
					t.Fatalf("validateDNSAccount01(%q) = success, but want error %q", tc.ident.Value, tc.wantErrMsg)
```